src/main/java/models/CustomerModelUtils.java

```diff
@@ -12,10 +12,13 @@ public static CustomerModel generateCustomerWithRandomData() {
         return CustomerModel.builder()
                 .firstName(fakerEN.name().firstName())
                 .lastName(fakerEN.name().lastName())
-                .email(fakerEN.name().lastName())
-                .address(fakerEN.address().streetAddress())
+                .email(fakerEN.internet().emailAddress())
+                .phone(fakerEN.phoneNumber().cellPhone())
+                .fax(fakerEN.phoneNumber().cellPhone())
+                .company(fakerEN.company().name())
+                .address1(fakerEN.address().streetAddress())
+                .address2(fakerEN.address().streetAddress())
                 .city(fakerEN.address().cityName())
-                .state(fakerEN.address().stateAbbr())
```

Comment:
Why state is removed?

Reply:
```state``` now has a default value because ```java faker``` does not generate a state depending on the country which caused errors when filling out the registeration form 
```java
    @Builder.Default
    public final String countryDefault = "United Kingdom"; // You can set the default country here
    public final String stateDefault = "Aberdeen";
```